### ISSUE TITLE
Unable to use PicoCMS with nextcloud 13 snap

### ISSUE THREAD

**Issue by skollrc** [nextcloud/nextcloud-snap#638]
Hi, 

I wanted to try PicoCMS but due to read only permission it's impossible to modify http.com. Is it possible to add correct conf as mentined here: https://github.com/nextcloud/cms_pico/wiki/Admin-Settings#apache-configuration ? 

Or can anyone tel me how to modifiy x-apache.py (or other file) to insert this config and rebuild a snap myself? 

Best regards

**Comment by JamesBream** [comment]
Take a look at the snap developer docs in the wiki, there's a guide on building your own custom snaps https://github.com/nextcloud/nextcloud-snap/wiki/Building-your-first-Nextcloud-snap

**Comment by skollrc** [comment]
I don't want to know how to create my own snap but just want to know what to do to be able to use PicoCMS app installed in a snap and if I need to rebuild or not

**Comment by pachulo** [comment]
@skollrc Yes, you are gonna need to build (aka "rebuild") a new snap, modifying the code from the original one following the instructions that @JamesBream pointed you to.

**Comment by skollrc** [comment]
Ok, thx for reply. My question is still the same. What do I have to modify and how ?

**Comment by ashgoodman** [comment]
So basically now way to use picocms with a snap install of nextcloud unless you are willing to roll your own?

The price of convenience I guess :)
